src/taoensso/timbre/appenders/3rd_party/newrelic.clj

```diff
@@ -0,0 +1,21 @@
+(ns taoensso.timbre.appenders.3rd-party.appenders.newrelic
+  "A New Relic appender"
+  {:author "Camilo Polymeris"}
+  (:import (com.newrelic.api.agent NewRelic)
+           (java.util HashMap)))
+
+(defn newrelic-appender
+  "New Relic appender. The Java agent must be installed and configured via newrelic.yml, see
+  https://docs.newrelic.com/docs/agents/java-agent/installation/java-agent-manual-installation#h2-install-agent"
+  [& _]
+  {:enabled?   true
+   :async?     true
+   :min-level  :warn
+   :rate-limit [[100 60000]]                                ;matches New Relic's own cap
+   :output-fn  :inherit
+   :fn         (fn [data]
+                 (let [{:keys [output-fn ?err level]} data
+                       params (HashMap. {"Log level" (name level)})]
+                   (if ?err
+                     (NewRelic/noticeError ?err params)
+                     (NewRelic/noticeError (output-fn data) params))))})
```

Comment:
Regarding the first question, I understand your point, but I don't see how I could not want all errors to be recorded by New Relic. Or conversly, if there were things I did not want sent to New Relic, I'd argue they are not errors, and I would be abusing `timbre/error` by logging them with it. (I'll need to think about this point, tho)
Slack isn't a logging app either, but I still want my errors to be sent there, too.

Regarding the second question... I don't know. If you are not convinced, maybe we can just close this PR. We can always reopen it later if the need for this appender becomes evident. Thank you for your patience.